WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+Reminders.swift

```diff
@@ -4,8 +4,7 @@ import SwiftUI
 extension BlogDetailsViewController {
```

Comment:
I see there's an `import SwiftUI` in this file that's no longer needed. Can we remove it?